atlas/commands/bootstrap/templates/docker/Dockerfile.gotmpl

```diff
@@ -9,4 +9,5 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o bin/server ./cmd/server
 FROM alpine:latest AS runner
 WORKDIR /bin
 COPY --from=builder /go/src/{{ .Root }}/{{ .Name }}/bin/server .
-ENTRYPOINT ["server"]
\ No newline at end of file
+{{ if .WithGateway }}COPY pkg/pb/service.swagger.json /go/src/{{ .Root }}/{{ .Name }}/www/swagger.json{{ end }}
```

Comment:
Actually, this isn't quite right. The second argument to the `COPY` command shouldn't be `/go/src/...` because that's a Go-specific convention (but it's running inside an Alpine container where Go isn't installed). Instead, it should the `swagger.json` should probably live in `/tmp/www/` or similar.